src/server/pps/server/api_server.go

```diff
@@ -972,6 +972,12 @@ func job(jobInfo *persist.JobInfo) *extensions.Job {
 								Privileged: &trueVal, // god is this dumb
 							},
 							ImagePullPolicy: "IfNotPresent",
+							Env: []api.EnvVar{
+								api.EnvVar{
+									Name:  "PACH_COMMIT_ID",
+									Value: jobInfo.OutputCommit.ID,
+								},
+							},
```

Comment:
I figure we might as well include the input commits as well while we're at it. Maybe as `PACH_OUT_COMMIT_ID` or something of that ilk.